src/DssSpell.sol

```diff
@@ -19,49 +19,36 @@ pragma solidity 0.8.16;
 import "dss-exec-lib/DssExec.sol";
 import "dss-exec-lib/DssAction.sol";
 
-interface RwaLiquidationLike {
-    function ilks(bytes32) external view returns (string memory doc, address pip, uint48 tau, uint48 toc);
-    function init(bytes32 ilk, uint256 val, string memory doc, uint48 tau) external;
-    function bump(bytes32 ilk, uint256 val) external;
-}
-
-interface RwaOutputConduitLike {
-    function hope(address usr) external;
-    function nope(address usr) external;
-    function mate(address usr) external;
-    function hate(address usr) external;
-    function kiss(address who) external;
-    function file(bytes32 what, address data) external;
-}
-
-interface RwaUrnLike {
-    function file(bytes32 what, address data) external;
-}
-
-interface NetworkPaymentAdapterLike {
-    function file(bytes32 what, address data) external;
+interface VestLike {
+    function file(bytes32 what, uint256 data) external;
+    function create(address, uint256, uint256, uint256, uint256, address) external returns (uint256);
+    function restrict(uint256) external;
+    function yank(uint256) external;
 }
 
 interface GemLike {
     function transfer(address, uint256) external returns (bool);
+    function approve(address, uint256) external returns (bool);
+    function allowance(address, address) external view returns (uint256);
+}
+
+interface ProxyLike {
+    function exec(address target, bytes calldata args) external payable returns (bytes memory out);
 }
 
 contract DssSpellAction is DssAction {
     // Provides a descriptive tag for bot consumption
     // This should be modified weekly to provide a summary of the actions
     // Hash: cast keccak -- "$(wget 'https://raw.githubusercontent.com/makerdao/community/e4bf988dd35f82e2828e1ce02c6762ddd398ff92/governance/votes/Executive%20vote%20-%20June%2028%2C%202023.md' -q -O - 2>/dev/null)"
     string public constant override description =
-        "2023-06-28 MakerDAO Executive Spell | Hash: 0x79a176bb631e7877acbdca1253e29354aa8fd4e3276dfd503fb3cd43f07d4fcd";
+        "2023-07-12 MakerDAO Executive Spell | Hash: TODO";
 
-    address internal immutable MIP21_LIQUIDATION_ORACLE       = DssExecLib.getChangelogAddress("MIP21_LIQUIDATION_ORACLE");
-    address internal immutable MCD_PSM_GUSD_A                 = DssExecLib.getChangelogAddress("MCD_PSM_GUSD_A");
-    address internal immutable RWA015_A_URN                   = DssExecLib.getChangelogAddress("RWA015_A_URN");
-    address internal immutable RWA015_A_OUTPUT_CONDUIT_LEGACY = DssExecLib.getChangelogAddress("RWA015_A_OUTPUT_CONDUIT");
-    address internal immutable MCD_ESM                        = DssExecLib.esm();
-    GemLike internal immutable MKR                            = GemLike(DssExecLib.mkr());
+    // Set office hours according to the summary
+    function officeHours() public pure override returns (bool) {
```

Comment:
I personally would prefer to keep it since it's more explicit. This way it's harder to forget to put back during clean up (which already happened in the previous spell)

Reply:
+1 on keeping it. Explicit is better than implicit.